xbmc/utils/SeekHandler.cpp

```diff
@@ -298,9 +307,102 @@ bool CSeekHandler::OnAction(const CAction &action)
         Seek(action.GetID() == ACTION_ANALOG_SEEK_FORWARD, action.GetAmount(), action.GetRepeat(), true);
       return true;
     }
+    case REMOTE_0:
+    case REMOTE_1:
+    case REMOTE_2:
+    case REMOTE_3:
+    case REMOTE_4:
+    case REMOTE_5:
+    case REMOTE_6:
+    case REMOTE_7:
+    case REMOTE_8:
+    case REMOTE_9:
+    {
+      if (!g_application.CurrentFileItem().IsLiveTV())
```

Comment:
If I'm not wrong a window handle this upfront, so we can remove this to do seeking for recordings. I can't check this right now, but will do asap.